cypress/tests/app.cy.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright The Cryostat Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { checkErrors } from '../support';
+
+export const isLocalDevEnvironment = Cypress.config('baseUrl')?.includes('localhost');
```

Comment:
Is this used, and if so what for? Does it need to be exported?

Reply:
Ah I'll remove that, it's not used at the moment.

In the template test it was used in the `before()` to setup helm for the test run. It might be nice to get back there eventually for a larger scale e2e run, but the idea is that the tests could start from an "empty" OpenShift environment and then install the test plugin using helm, and then uninstall it once the tests complete.